qlib/kernel/arch/aarch64/mm/pagetable.rs

```diff
@@ -119,9 +120,13 @@ bitflags! {
     pub struct PageTableFlags: u64 {
         const VALID           = 1;
         const TABLE           = 1 << 1;
-        const USER_ACCESSIBLE = 1 << 6;
+        const PAGE            = 3;      // 4k granule
+        const DEVICE          = 1 << 2; // device mmio
```

Comment:
This flag DEVICE is same as MT_DEVICE_NGNRE below.
remove duplicates.
I suggest u rename the two to: MEM_DEVICE and MEM_NORMAL, for device memory and normal memory respectively.

Reply:
(TODO/suggestion) multiple-bit flags should be avoided for 
some fields e.g. Memory Attributes see  
https://docs.rs/bitflags/latest/bitflags/#multi-bit-flags

Reply:
That's the meaning of each of those flag. It does not imply multiple bits